### DISCUSSION TITLE
How to access the PostgREST API data in prod.

### DISCUSSION THREAD

**Discussion by femakin** [PostgREST/postgrest#3048] [Q&A] [answered]
Good evening, everyone.

I am currently delving into PostgREST, and I have successfully established a connection between my database and PostgREST, enabling me to integrate the API into my Next.js application. However, I seek guidance on the proper procedures for deploying PostgREST and my database to a production environment and hosting platform such as Render. Any insights or recommendations on this matter would be greatly appreciated. Thank you.

I can access the PostgREST data locally using [localhost:3000](http://localhost:3000/). But this won't work in prod. My App is deployed to render. Please how do I go further from here ?

**Comment by laurenceisla** [comment, marked as answer]
Hi! I was going to suggest using Nginx [as explained in this tutorial](https://www.youtube.com/watch?v=TfvmfrGD5EQ&t=642s), but it seems that Render has its own way of deploying applications. According to Render, here's [how to deploy a PostgreSQL DB](https://render.com/docs/databases) and here's an example I found for a [PostgREST deployment](https://github.com/render-examples/postgrest/tree/main) (it uses the Docker image).

You'll have to experiment a bit there, I'm not sure if it works or if it's safe/the right way to deploy it. I suggest also asking the Render support to get more info on the deployment process.
